nmagent/internal/wireserver_test.go

```diff
@@ -247,8 +252,9 @@ func TestWireserverTransportPostBody(t *testing.T) {
 	}
 	defer resp.Body.Close()
 
-	if bodyIsNil {
-		t.Error("downstream request body to wireserver was nil, but not expected to be")
+	exp := `""` // an empty JSON string
+	if got := string(gotBody); got != exp {
+		t.Errorf("unexpected body received: got: %q, exp: %q\n", got, exp)
```

Comment:
nit: is the new line needed? same for others.

Reply:
@ramiro-gamarra do you mean the changed content, or a literal `\n`? There are no extra `\n`s (look at the line number sequence). It might be a GH display bug.

Reply:
the `\n` in the error message

Reply:
Okay, so apparently, the presence or absence of it makes no difference in the output--they're stripped internally. I add them out of habit it because it matters for `fmt.Println` vs `fmt.Printf`. Since it won't create janky output, I'll keep it in mind for next time, but I won't fix it here to avoid blocking the merge.